### PR TITLE
closes #533

### LINKED ISSUE #533: Mention HTML unit in file downloads section
## 🚀 Feature Proposal for the site or the documentation

https://www.selenium.dev/documentation/en/worst_practices/file_downloads/

HTMLunit has a mature interface for file downloads ( see https://htmlunit.sourceforge.io/apidocs/com/gargoylesoftware/htmlunit/attachment/AttachmentHandler.html )

It could be mentioned as many are looking to use Selenium for web scraping and file downloads and copying a link is not always possible, especially with more complex JavaScript sites.

Not sure if there is a better way to handle file downloads via Selenium currently that to use HTMLunit, it's driver and native input streams

## Motivation

Took me quite a bit of time to figure out how to download files. Having at least a mention in the official docs would probably hell others save time.

## Example

HtmlUnit and its driver can be used to intercept files as input streams and add custom logic that will handle and save them.

See https://github.com/SeleniumHQ/htmlunit-driver and https://github.com/HtmlUnit/htmlunit/blob/master/src/main/java/com/gargoylesoftware/htmlunit/attachment/AttachmentHandler.java

### PR DESCRIPTION
**Pull request by skyhirider**
described attachment download option via HtmlUnit

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
closes #533 
### Motivation and Context

as mentioned in issue, a lot of people are looking to download files from ajax sites and selenium + htmlunit is one of the few options available if you asynchronously receive a download response
### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by harsha509 — DISMISSED** [review]
hi @skyhirider !

Thanks for the PR !

Can you please make same changes in other translated pages too (like es, zh-cn etc)

**Comment by skyhirider** [comment]
@Harsha509 You mean translate it? Best I could do is run it through google translate, did you mean that?

My native tongue is Slovak and have a limited grasp of German, and that is pretty much it.

**Comment by diemol** [comment]
@skyhirider I replied here
https://github.com/SeleniumHQ/seleniumhq.github.io/issues/533#issuecomment-708016220

**Comment by harsha509** [comment]
Hi @skyhirider ,

Did you get a chance to look into the above comments/suggestions ?

**Comment by skyhirider** [comment]
Hi @JS31096 I will have a look, still want to make the change just need to find time to do it.

**Comment by skyhirider** [comment]
@JS31096 added English text to other language versions with the disclaimer. Also ticked the checkbox for the CLA

File: docs_source_files/content/worst_practices/file_downloads.es.md [hunk 1]
```diff
@@ -10,4 +10,12 @@ Esto es debido a que descargar archivos no es considerado un aspecto importante
 de la emulación de las interacciones de los usuarios con las plataformas web.
 En lugar de ello, se recomienda encontrar el enlace con Selenium (y cualquier
 Cookie requerida) y pasarselo a una librería que permita hacer peticiones HTTP 
-como [libcurl](//curl.haxx.se/libcurl/).
\ No newline at end of file
+como [libcurl](//curl.haxx.se/libcurl/).
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.es.md]:
```suggestion
English to Spanish. Do you speak Spanish? Help us to translate
```

File: docs_source_files/content/worst_practices/file_downloads.fr.md [hunk 2]
```diff
@@ -14,4 +14,12 @@ d'émuler l'interaction des utilisateurs avec la plate-forme Web.
 À la place, recherchez le lien à l'aide de Selenium
 (et tous les cookies requis)
 et le passer à une bibliothèque de requêtes HTTP comme
-[libcurl](//curl.haxx.se/libcurl/).
\ No newline at end of file
+[libcurl](//curl.haxx.se/libcurl/).
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.fr.md]:
```suggestion
English to French. Do you speak French? Help us to translate
```

File: docs_source_files/content/worst_practices/file_downloads.ja.md [hunk 3]
```diff
@@ -7,3 +7,11 @@ weight: 2
 Seleniumの管理下にあるブラウザーでリンクをクリックしてダウンロードを開始することは可能ですが、APIはダウンロードの進行状況を公開しないため、ダウンロードしたファイルのテストには理想的ではありません。
 これは、ファイルのダウンロードは、Webプラットフォームとのユーザーインタラクションをエミュレートする重要な側面とは見なされないためです。
 代わりに、Selenium（および必要なCookie）を使用してリンクを見つけ、 [libcurl](//curl.haxx.se/libcurl/) などのHTTPリクエストライブラリに渡します。
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.ja.md]:
```suggestion
English to Japanese. Do you speak Japanese? Help us to translate
```

File: docs_source_files/content/worst_practices/file_downloads.ko.md [hunk 4]
```diff
@@ -6,3 +6,11 @@ weight: 2
 Selenium의 제어 하에 브라우저로 링크를 클릭하면 다운로드를 시작할 수 있지만, API는 다운로드 진행 상황을 노출시키지 않아 다운로드 파일 테스트에 이상적이지 않다.
 이것은 파일 다운로드가 웹 플랫폼과의 사용자 상호작용을 모방하는 중요한 측면으로 여겨지지 않기 때문이다.
 대신 Selenium(및 필요한 쿠키)을 사용하여 링크를 찾아 HTTP 요청 라이브러리 like [libcurl](//curl.haxx.se/libcurl/).에 전달하라.
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.ko.md]:
```suggestion
English to Korean. Do you speak Korean? Help us to translate
```

File: docs_source_files/content/worst_practices/file_downloads.nl.md [hunk 5]
```diff
@@ -18,4 +18,12 @@ of emulating user interaction with the web platform.
...
 and pass it to a HTTP request library like
-[libcurl](//curl.haxx.se/libcurl/).
\ No newline at end of file
+[libcurl](//curl.haxx.se/libcurl/).
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
+it by sending us pull requests!
+{{% /notice %}}
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.nl.md]:
```suggestion
```

File: docs_source_files/content/worst_practices/file_downloads.zh-cn.md [hunk 6]
```diff
@@ -6,3 +6,11 @@ weight: 2
 
 虽然可以通过在Selenium的控制下单击浏览器的链接来开始下载, 但是API并不会暴露下载进度, 因此这是一种不理想的测试下载文件的方式.
 因为下载文件并非模拟用户与Web平台交互的重要方面. 取而代之的是, 应使用Selenium(以及任何必要的cookie)查找链接, 并将其传递给例如[libcurl](//curl.haxx.se/libcurl/)这样的HTTP请求库.
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Page being translated from 
+English to {LANGUAGE}. Do you speak {LANGUAGE}? Help us to translate
```
Comment (diemol) [inline docs_source_files/content/worst_practices/file_downloads.zh-cn.md]:
```suggestion
English to Chinese. Do you speak Chinese? Help us to translate
```

**Review by diemol — APPROVED** [review]

**Pull request merged**